project/Helpers.scala

```diff
@@ -224,18 +224,18 @@ object C {
     dockerRepository := sys.props.get("docker.registry"),
     dockerUpdateLatest := true,
     dockerExposedPorts ++= Seq(port),
-    dockerBuildCommand := {
-      if (sys.props("os.arch") != "amd64") {
-        // use buildx with platform to build supported amd64 images on other CPU architectures
-        // this may require that you have first run 'docker buildx create' to set docker buildx up
-        dockerExecCommand.value ++ Seq(
-          "buildx",
-          "build",
-          "--platform=linux/amd64",
-          "--load"
-        ) ++ dockerBuildOptions.value :+ "."
-      } else dockerBuildCommand.value
-    }
+    //dockerBuildCommand := {
```

Comment:
What's up with this commented-out section? (Please make a comment as to why it's commented out or just remove it if it's no longer needed)

Reply:
I uncommented it because it is causing errors for me all over the place (local tests & experiments with microk8s). Not sure anyway why we would want to compile for amd64 when we have no deployment architecture. Will leave a note for developers if they want to turn it back on.